deployment/maintain/rebase-helper.md

```diff
@@ -52,34 +55,34 @@ Rebase-helper provides an API which you can use either directly from Python, or
 Example of patching new sources and starting scratch builds with fedpkg.
  This returns task_ids. The bash equivalents are included for comparison:
  
-* ``Python API``
+* **Python API**
 
-```
-from rebasehelper.application import Application
-cli = CLI([‘--non-interactive, ‘--builds-nowait’, ‘-buildtool’, ‘fedpkg’, upstream_version])
-rh = Application(cli)
-rh.set_upstream_monitoring() # Switch rebase-helper to upstream release monitoring mode.
-rh.run()
-```
+   ```python3
+   from rebasehelper.application import Application
+   cli = CLI(['--non-interactive', '--builds-nowait', '--buildtool', 'fedpkg', 'upstream_version'])
+   rh = Application(cli)
+   rh.set_upstream_monitoring() # Switch rebase-helper to upstream release monitoring mode.
+   rh.run()
+   ```
 
-* ``BASH``
+* **BASH**
 
-```
-rebase-helper --non-interactive --builds-nowait --buildtool fedpkg upstream_version
-```
+   ```sh
+   rebase-helper --non-interactive --builds-nowait --buildtool fedpkg upstream_version
+   ```
 
-### Download logs and RPMs and compare with tools like abipkgdiff
+### Download logs and RPMs and compare with tools like ``abipkgdiff``
 
-* ``Python API``
+* **Python API**
 
-```
-cli = CLI([‘--non-interactive, ‘--builds-nowait’, ‘--fedpkg-build-tasks old_id,new-id])
-rh.run() # Downloads RPMs, logs and runs checkers and provides logs.
-rh.get_rebasehelper_data() # Get all information about the results
-```
+   ```python3
+   cli = CLI(['--non-interactive', '--builds-nowait', '--buildtool', 'fedpkg', '--build-tasks', 'old_id,new_id'])
+   rh.run() # Downloads RPMs, logs and runs checkers and provides logs.
+   rh.get_rebasehelper_data() # Get all information about the results
+   ```
 
-* ``BASH``
+* **BASH**
```

Comment:
Wouldn't `command line` be more accurate here?

Reply:
I don't... _think_ so. My impression is that the examples there were meant for `/bin/sh` shell scripting, when integrating into a monitoring service controlled by shell scripts. They weren't envisioned as interactive commands. But, that's purely my guess of the original author's intent, based on the context.

Reply:
(In fact, while I added `$` prompts to the start of these commands, too, I'm now thinking that may have been a mistake, for exactly those reasons.)